pkg/apis/build/v1beta1/buildrun_types.go

```diff
@@ -103,6 +103,10 @@ const (
 	// if not already canceled or terminated
 	BuildRunStateCancel = "BuildRunCanceled"
 
+	// BuildRunStateVulnerabilitiesFound indicates that the user wants to fail the buildrun,
+	// if vulnerabilities in the image scanning are found
+	BuildRunStateVulnerabilitiesFound = "VulnerabilitiesFound"
```

Comment:
`BuildRunCanceled` works a little different than the other states that we have here: you set it on BuildRun.spec.state as desired state when you want to cancel a running BuildRun. Therefore the "wants to" indicating that something should happen. Your state is purely to describe what had happened.

```suggestion
	// BuildRunStateVulnerabilitiesFound indicates that the unignored vulnerabilities were found in the image that was built
	BuildRunStateVulnerabilitiesFound = "VulnerabilitiesFound"
```